src/analysis/souffle/analyze_tag_checks_test_helper.dl

```diff
@@ -0,0 +1,74 @@
+#ifndef SRC_ANALYSIS_SOUFFLE_ANALYZE_TAG_CHECKS_TEST_HELPER_DL_
+#define SRC_ANALYSIS_SOUFFLE_ANALYZE_TAG_CHECKS_TEST_HELPER_DL_
+
+#include "taint.dl"
+#include "analyze_tag_checks.dl"
+
+// This is a tag check that does not correspond to the original test's arcs manifest. We want to
+// eventually translate the arcs manifests into facts and check them against our hand-written facts,
+// so distinguishing the extra tags will be important eventually. For now, thought, it just works
+// exactly like checkTag.
+.decl checkTagExtra(accessPath: AccessPath, tag: Tag, tagPresence: TagPresence)
+
+checkTag(accessPath, tag, tagPresence) :- checkTagExtra(accessPath, tag, tagPresence).
+
+.type TestResult <: symbol
+.decl isValidTestResult(testResult: TestResult)
+
+isValidTestResult("PASS").
+isValidTestResult("FAIL").
+
+.decl tagCheckExpectation(
```

Comment:
What does this relation capture?

Reply:
No longer exists, is gone after I performed the simplification you proposed.